Sources/XcodesKit/FastlaneCookieLoader.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  FastlaneCookieLoader.swift
+//  
+//
+//  Created by Omar Zuñiga on 09/01/23.
+//
+
+import Foundation
+import Yams
+
+struct FastlaneCookieLoader {
+    
+    private let fastlaneSession = "FASTLANE_SESSION"
+    
+    func load(in storage: HTTPCookieStorage?) {
+        do {
+            guard let storage, var sessionVar = Current.shell.env(fastlaneSession) else { return }
```

Comment:
I think it's also valuable to automatically load the cookie if it is present in the filesystem. fastlane stores it in the file `~/.fastlane/spaceship/email@domain.com/cookie`.